run_project_tests.py

```diff
@@ -1280,6 +1280,15 @@ def tqdm_print(*args: mlog.TV_Loggable, sep: str = ' ') -> None:
             safe_print(bold('During:'), result.step.name)
             safe_print(bold('Reason:'), result.msg)
             failing_tests += 1
+            # Append a visual seperator for the different test cases
+            cols = shutil.get_terminal_size((100, 20)).columns
+            name_str = ' '.join([str(x) for x in f.testdef.display_name()])
+            name_len = len(re.sub(r'\x1B[^m]+m', '', name_str))  # Do not count escape sequences
+            left_w = (cols / 2) - (name_len / 2) - 1
```

Comment:
instead of doing the int case below you probably want to use floor division (`//`) here

Reply:
Thanks. Still learning new things in Python :)